### ISSUE TITLE
t - customAttribute and params

### ISSUE THREAD

**Issue by Mordred** [aurelia/i18n#9]
It is possible to somehow pass parameters to translation when using attribute?

E.g.:
```html
<span t="[html]translation.with.params" t-params.bind="params">
```

**Comment by zewa666** [comment]
I'm not sure I fully understand what you'd like to achieve. Can you show me what you'd do in code and what the translation key would look like?

**Comment by Mordred** [comment]
I need to pass some arguments to translation:

```json
{
  "translations": {
    "with": {
      "params": "Some text with <strong>__content__</strong>"
    }
  }
}
```

```js
params = { content: 'NICE' }
```

Currently this is possible only as:

```html
<span>${'translation.with.params' | t: params | sanitizeHtml }</span>
```

I would be nice to have some option how to pass parameters to "t" attribute.

**Comment by zewa666** [comment]
Nope currently not, but thanks for the good idea :+1:

**Comment by zewa666** [comment]
@Mordred could you give it now a try, the feature has now been added and should work as expected. Funny enough another issue with having values bound to t `t.bind = ...` now is fixed as well :)

**Comment by valichek** [comment]
@zewa666 hi, I just tried to use ```t.bind = ...``` but doesn't work for me, I use ```github:aurelia/i18n@0.2.2```, it binds only once, and then when locale is switched nothing happens

**Comment by zewa666** [comment]
@valichek could you show me the markup and vm as well as the two translations you've used for the test?

**Comment by valichek** [comment]
here is markup
```html
<template>
  <ul class="steps">
    <li repeat.for="step of wizard.steps" class="${step.isActive? 'active' : ''}">
      <span class="step" click.trigger="$parent.wizard.tryActivate(step)">${step.num}</span>
      <span t.bind="step.title" class="title"></span>
    </li>
  </ul>
</template>
```
`step.title` contains the key in translation smth like: `"wizard-steps.0"`
And translation: 
```json 
{"wizard-steps" : ["General", "Target Group", "Promotion", "Overview", "Start"]}
```

**Comment by zewa666** [comment]
Ok I forgot to include the event listener for locale changes, there is a new version just now released 0.2.3 which should fix that. @valichek please try it out again with that one.

**Comment by valichek** [comment]
Now works like a charm) Thx

**Comment by zewa666** [comment]
thx @valichek, now that you reviewed this we only need to wait for @Mordred's reply on testing the params then we may close this issue

**Comment by Mordred** [comment]
@zewa666 Sorry for delay. `t-params` seems to be working only if you define it before `t` attribute:

This works:
```html
<p t-params.bind="config" t="[html]user.people._modals.createCinepass.instructions"></p>
```

This doesn't work:
```html
<p t="[html]user.people._modals.createCinepass.instructions" t-params.bind="config"></p>
```

You should warn about this in the documentation.

**Comment by valichek** [comment]
hm, for me that works
```html
<div repeat.for="item of model.items" class="widget-box transparent">
   <h5 class="widget-title" t.bind="'actions.communication-title'" t-params.bind="item"></h5>
</div>
```
But if I place `t-params.bind=` before `t.bind=`, it doesn't

**Comment by valichek** [comment]
and it look like my last example works for Firefox, and not for Safari and Chrome

**Comment by zewa666** [comment]
ok @valichek and @Mordred. Your both input and testing so far was invaluable, so a big thanks to both of you. The recent release now should fix that issue with a new custom resolver. So whenever you have time give it another try. Thanks in advance

**Comment by Mordred** [comment]
@zewa666 Good job, it is working now.
